src/mapInteractor.js

```diff
@@ -64,28 +55,66 @@ var mapInteractor = function (args) {
     {
       throttle: 30,
       discreteZoom: false,
-      panMoveButton: 'left',
-      panMoveModifiers: {},
-      zoomMoveButton: 'right',
-      zoomMoveModifiers: {},
-      rotateMoveButton: 'left',
-      rotateMoveModifiers: {ctrl: true},
-      panWheelEnabled: false,
-      panWheelModifiers: {},
-      zoomWheelEnabled: true,
-      zoomWheelModifiers: {},
-      rotateWheelEnabled: true,
-      rotateWheelModifiers: {ctrl: true},
+
+      /* There should only be one action with any specific combination of event
+       * and modifiers.  When that event and modifiers occur, the specified
+       * action is triggered.  The event and modifiers fields can either be a
+       * simple string or an object with multiple entries with each entry set
+       * to true, false, or undefined.  If an object, all values that are
+       * truthy must match, all values that are false must not match, and all
+       * other values that are falsy are ignored.
+       *   Available actions:
+       * pan, zoom, rotate, select, zoomselect, unzoomselect, click
+       *   Available events:
+       * left, right, middle, wheel
+       *   Available modifiers:
+       * shift, ctrl, alt, meta
+       */
+      actions: [{
+        action: 'pan',
```

Comment:
@manthey should we define action and event values as some keys at one single place? for e.g. (or some other way)

user_actions = {
pan: "pan",
zoom: "zoom"
}

and then we can have: 

action: user_actions.pan

Thoughts?